TAGradingServer/toolbox/include/custom/js/diff-viewer.js

```diff
@@ -28,4 +28,19 @@ $(document).ready(function() {
     };
 
     $('.highlight').hover(hoverOn, hoverOff);
+
+    // Check if document.getElementsByClassName is supported
+    if (!document.getElementsByClassName)
+        return;
```

Comment:
whatever browser doesn't support getElementsByClassName is a browser we don't care about (according to caniuse.com, it's IE 9- and FF 2 that don't support this and who the hell is still using those at university?).

Reply:
@LeonMontealegre please remove these checks in your next commit.  I'm going to merge this PR now.